docs/_docs/contributing/index.md

```diff
@@ -39,11 +40,3 @@ another programming language.
 As an example, `dotc` takes text input, verifies that it is a valid Scala program
 and then produces as output the same program, but written in Java bytecode, and optionally
 in SJSIR when producing Scala.js output.
-
-### Contribute Internals-related Knowledge
-If you know anything useful at all about Dotty, feel free to log this knowledge:
-
-- [📜Log the Knowledge](https://github.com/lampepfl/dotty-knowledge/issues/new)
```

Comment:
So this is something I'd like to get your thoughts on @anatoliykmetyuk. I've never looked here before, and this seems to have gone dormant over the years and many of the things listed there are either stale or somewhat ported here to here. If you're ok with it, I'd like to archive that repo, make sure everything in the issues over there is actually in this guide, and focus on making sure all the knowledge is in one place. wdyt?

Reply:
Just confirming that I did check with @anatoliykmetyuk about this and I'll make sure to go through it all and make sure we didn't miss anything from there.